routers/api/v1/repo/pull.go

```diff
@@ -284,6 +284,12 @@ func CreatePullRequest(ctx *context.APIContext, form api.CreatePullRequestOption
 	//   "422":
 	//     "$ref": "#/responses/validationError"
 
+	if form.Head == form.Base {
+		ctx.Error(http.StatusUnprocessableEntity, "BaseHeadSame",
+			"Invalid PullRequest: No changes presented. ( Head is equal to Base )")
```

Comment:
```suggestion
			"Head and Base are now allowed to be the same")
```

Reply:
@techknowlogick do you have a better one :point_up: ?

Reply:
```suggestion
			"Invalid PullRequest: There are no changes between the head and the base")
```